types/src/crypto_proxies.rs

```diff
@@ -77,13 +79,51 @@ impl<Sig: RawSignature> From<Sig> for SignatureWrapper<Sig> {
 // below is banned.
 
 use crypto::ed25519::*;
+use std::collections::HashMap;
 
 // used in chained_bft::consensus_types::block_test
 #[cfg(any(test, feature = "testing"))]
 pub type SecretKey = Ed25519PrivateKey;
 
 pub type Signature = SignatureWrapper<Ed25519Signature>;
 pub type LedgerInfoWithSignatures = RawLedgerInfoWithSignatures<Ed25519Signature>;
+pub type ValidatorInfo = RawValidatorInfo<Ed25519PublicKey>;
 pub type ValidatorVerifier = RawValidatorVerifier<Ed25519PublicKey>;
 pub type ValidatorSigner = RawValidatorSigner<Ed25519PrivateKey>;
 pub type ValidatorChangeEventWithProof = RawValidatorChangeEventWithProof<Ed25519Signature>;
+
+/// Helper function to get random validator signers and a corresponding validator verifier for
+/// testing.  If custom_voting_power_quorum is not None, set a custom voting power quorum amount.
+/// With pseudo_random_account_address enabled, logs show 0 -> [0000], 1 -> [1000]
```

Comment:
Note: we probably will want an `Arbitrary` instance for that as well.

Reply:
Future work?  The PR doesn't require it right?

Reply:
No, but it would look nice alongside `signer_strategy` in `consensus/src/chained_bft/consensus_types/block_test.rs`

Reply:
Good idea.  Let's save it for a future PR.